### ISSUE TITLE
How to train GCN?

### ISSUE THREAD

**Issue by mrlzla** [ycszen/pytorch-ss#7]
How to train my own dataset on GCN?
Is there some best practices to train the model?
Learning rate multiplication gets better accuracy?
